ovs/extensions/healthcheck/generic_hc.py

```diff
@@ -86,6 +86,27 @@ def get_log_files_by_path(start_path, recursive=True):
             result_handler.success('All log files are ok!')
 
     @staticmethod
+    @expose_to_cli('ovs','port-ranges-test',HealthCheckCLIRunner.ADDON_TYPE)
+    def check_port_ranges(result_handler, requested_number_of_ports=2, minimal_number_of_ports=2):
```

Comment:
Remark 1: I'd opt for parameter names: `requested_ports` and `expected_ports` to make them shorter and even meaningful

Remark 2: Default value of 2 is not sufficient for in a healthcheck scenario because when deploying a StorageDriver, we require 4 ports to be available (XMLRPC, DTL, Edge, Storage)

Remark 3: Bad function name, since only the StorageDriver range is being checked for, while we have other ranges too (MDS, Arakoon, ...)